apps/condo/domains/user/integration/telegram/constants.js

```diff
@@ -0,0 +1,18 @@
+const TELEGRAM_AUTH_STATUS_PENDING = 'pending'
+const TELEGRAM_AUTH_STATUS_SUCCESS = 'success'
+const TELEGRAM_AUTH_STATUS_ERROR = 'error'
+
+const TELEGRAM_AUTH_REDIS_STATE_PREFIX = 'telegram:state:'
+
+const TELEGRAM_AUTH_REDIS_TTL = 180 // 3 mins
```

Comment:
Its common pattern to end duration variable with `_IN_<measure>`, for example `TELEGRAM_AUTH_REDIS_TTL_IN_SEC`

Reply:
So it will be clear, if comment is lost / value changed